modules/snooker/src/main/ts/ephox/snooker/resize/BarManager.ts

```diff
@@ -133,34 +140,42 @@ export const BarManager = (wire: ResizeWire): BarManager => {
         * This is fairly safe to do frequently; it's a single querySelectorAll() on the content and Arr.map on the result.
         * If we _really_ need to optimise it further, we can start caching the bar references in the wire somehow.
         */
-        if (SugarBody.inBody(event.target)) {
+        if (SugarBody.inBody(event.target) && !Class.has(event.target, resizerBar) && !isDragging.get()) {
           Bars.destroy(wire);
         }
       },
       (table) => {
         if (resizing.isActive()) {
-          hoverTable = Optional.some(table);
+          hoverTable.set(table);
+          events.trigger.hoverTable(hoverTable.get());
           Bars.refresh(wire, table);
         }
       }
     );
   });
 
   const destroy = () => {
-    mousedown.unbind();
-    mouseover.unbind();
     resizing.destroy();
     Bars.destroy(wire);
   };
 
+  const unbindEvents = () => {
```

Comment:
Why was this separated, having a destroy that doesn't unbind the events seems odd? Maybe `destroy` needs to be renamed `hide` or something and a new `destroy` just calls `hide` and `unbindEvents`. Destroy to me at least means remove everything about the resize bars, the DOM structure, the events etc. Like go back to what it was before we added it.